docs/docs/reference/other-new-features/opaques.md

```diff
@@ -81,7 +83,7 @@ object Access {
   val ReadOrWrite: PermissionChoice = Read | Write
 }
 ```
-The `Access` object defines three opaque types:
+The `Access` object defines three opaque type aliases:
```

Comment:
I wonder if we should go the other way and call them "opaque types" everywhere since "opaque type aliases" is a bit of mouthful and less commonly used in my experience. But any consistency is good!

Reply:
_Opaque type alias_ is surely a bit more precise but I would not mind to use opaque type as one has to look up what it means in any case I guess.. Other pages use opaque type already. I could do a search replace once we agree on one of the two (or something else).

Reply:
Just asked around at the lab and people seemed to prefer "opaque type aliases" so let's keep it that way :).